.github/actions/spelling/allow.txt

```diff
@@ -264,6 +264,7 @@ IPortable
 ipmo
 ISAPPROVEDFOROUTPUT
 isspace
+istavailable
```

Comment:
This GitHub spelling action appears to support patterns to ignore, so maybe instead of allowing these partial words, have it ignore anything within backticks?